test/game_statistics_test.rb

```diff
@@ -60,11 +60,17 @@ def test_average_goals_by_season
     game_statistics = GameStatistics.new(@game.stat_hash)
 
     assert_equal ({"20122013" => 3.68}), game_statistics.average_goals_by_season
+  end
 
   def test_average_goals_per_game
     game_statistics = GameStatistics.new(@game.stat_hash)
 
     assert_equal 3.68, game_statistics.average_goals_per_game
 
   end
+
+  def test_it_can_determine_percentage_of_visitor_wins
```

Comment:
I see that you have a test for the percentage_visitor_wins method. 

Thank you for ensuring that we perform proper TDD.